lib/iris/_representation/cube_printout.py

```diff
@@ -257,9 +257,12 @@ def add_scalar_row(name, value=""):
                         add_scalar_row(item.name, item.content)
                         if item.extra:
                             add_scalar_row(item_to_extra_indent + item.extra)
-                elif "attribute" in title or "cell method" in title:
+                elif "attribute" in title:
                     for title, value in zip(sect.names, sect.values):
                         add_scalar_row(title, value)
+                elif "cell method" in title:
+                    for title, value in zip(sect.names, sect.values):
+                        add_scalar_row(", ".join(title), value)
```

Comment:
```suggestion
                    for coords, method in zip(sect.names, sect.values):
                        add_scalar_row(", ".join(coords), method)
```

Specialised behaviour will benefit from specialised language to help understanding of what's going on.